src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTest.cs

```diff
@@ -637,5 +637,46 @@ public void Process_Environment()
             }
             );
         }
+
+        [Fact]
+        public void Process_StartInfo()
+        {
+            {
+                Process process = CreateProcessInfinite();
+                process.Start();
+
+                Assert.Equal(ProcessName, process.StartInfo.FileName);
+
+                process.Kill();
+                process.WaitForExit(WaitInMS);
+            }
+
+            {
+                Process process = CreateProcessInfinite();
+                process.Start();
+
+                Assert.Throws<System.InvalidOperationException>(() => (process.StartInfo = new ProcessStartInfo()));
+
+                process.Kill();
+                process.WaitForExit(WaitInMS);
+            }
+
+            {
+                Process process = new Process();
+                process.StartInfo = new ProcessStartInfo(ProcessName);
+                Assert.Equal(ProcessName, process.StartInfo.FileName);
+            }
+
+            {
+                Process process = Process.GetCurrentProcess();
+                Assert.Throws<System.InvalidOperationException>(() => process.StartInfo);
+            }
+
+            {
+                Process process = Process.Start(ProcessName);
```

Comment:
That said... how is this test different from the first one in the function?  Or more specifically, what's the difference you're testing between the two?